pyuvdata/uvdata/src/miriad_wrap.pyx

```diff
@@ -200,7 +200,7 @@ cpdef hread_init(int item_hdl) except +raise_miriad_error:
   else:
     raise RuntimeError("unknown item type.")
 
-cpdef hwrite(int item_hdl, int offset, val, str type) except +raise_miriad_error:
+cpdef int hwrite(int item_hdl, int offset, val, str type) except *:
```

Comment:
I don't know enough about cython to know what this means and why it should change.

Reply:
Typing returns helps optimize the hand off and reduce overhead between c calls in cython. This was also on my radar. Beat me to it 😂

Reply:
Great. What about the change from `+raise_miriad_error` -> `*`

Reply:
Ah yeah that's about how exceptions get propagated up to python. I'm making a bunch of changes like this in my branch too. The short of it is I don't think the raise_miriad_error did anything anyway

Reply:
Yeah, I ~stole~ borrowed some of this from #989. I had to get rid of the reference to `np.int` in the type checking, and included other changes for this function. And yes, the `raise_miriad_error` was just a `pass` statement, so I don't think it actually did anything.